api/lib/infrastructure/repositories/competence-repository.js

```diff
@@ -7,44 +7,52 @@ const competenceDatasource = require('../datasources/airtable/competence-datasou
 const knowledgeElementRepository = require('./knowledge-element-repository');
 const scoringService = require('../../domain/services/scoring/scoring-service');
 const { NotFoundError } = require('../../domain/errors');
+const { FRENCH_FRANCE, ENGLISH_SPOKEN } = require('../../domain/constants').LOCALE;
 
 const PixOriginName = 'Pix';
 
-function _toDomain(competenceData, areaDatas) {
+function _toDomain(competenceData, areaDatas, locale) {
```

Comment:
Est-ce qu'on ne devrait pas suivre le pattern [RORO](https://medium.com/free-code-camp/elegant-patterns-in-modern-javascript-roro-be01e7669cbd) ici ? 
```suggestion
function _toDomain({ competenceData, areaDatas, locale }) {
```